app.py

```diff
@@ -38,6 +43,23 @@ def get_stream():
         return streams[0]
     return '{}'
 
+@app.route('/streamtrainId/<trainId>')
+def get_stream_train(trainId):
+    print("Trainid: ", trainId)
+    streams = getStreams(trainId=trainId)
+    if streams:
+        return streams[0]
+    return '{}'
```

Comment:
why are we returning '{}',
can you please help me understand

Reply:
We return an empty json object (that is, a json object as it is represented in python as a string) whenever no stream with that specific ID is found. This will raise the pop up alert "Uh oh! There's been an issue retrieving a stream..." if it occurs when hitting a button.

Reply:
I was assuming to throw error and catch in FE
but this also make sense
Thanks